src/editors/containers/ProblemEditor/components/EditProblemView/AnswerWidget/AnswerOption.jsx

```diff
@@ -66,91 +27,56 @@ export const AnswerOption = ({
   const setAnswer = hooks.setAnswer({ answer, hasSingleAnswer, dispatch });
   const { isFeedbackVisible, toggleFeedback } = hooks.prepareFeedback(answer);
 
-  const displayFeedbackControl = (answerObject) => {
-    if (problemType !== ProblemTypeKeys.MULTISELECT) {
-      return FeedbackControl({
-        key: `feedback-${answerObject.id}`,
-        feedback: answerObject.selectedFeedback,
-        onChange: (e) => setAnswer({ selectedFeedback: e.target.value }),
-        labelMessage: messages.selectedFeedbackLabel,
-        labelMessageBoldUnderline: messages.selectedFeedbackLabelBoldUnderlineText,
-        answer: answerObject,
-        intl,
-      });
-    }
-    return [
-      FeedbackControl({
-        key: `selectedfeedback-${answerObject.id}`,
-        feedback: answerObject.selectedFeedback,
-        onChange: (e) => setAnswer({ selectedFeedback: e.target.value }),
-        labelMessage: messages.selectedFeedbackLabel,
-        labelMessageBoldUnderline: messages.selectedFeedbackLabelBoldUnderlineText,
-        answer: answerObject,
-        intl,
-      }),
-      FeedbackControl({
-        key: `unselectedfeedback-${answerObject.id}`,
-        feedback: answerObject.unselectedFeedback,
-        onChange: (e) => setAnswer({ unselectedFeedback: e.target.value }),
-        labelMessage: messages.unSelectedFeedbackLabel,
-        labelMessageBoldUnderline: messages.unSelectedFeedbackLabelBoldUnderlineText,
-        answer: answerObject,
-        intl,
-      }),
-    ];
-  };
-
   return (
     <Collapsible.Advanced
       open={isFeedbackVisible}
       onToggle={toggleFeedback}
-      className="collapsible-card"
+      className="answer-option collapsible-card d-flex flex-row justify-content-between flex-nowrap pb-2 pt-2"
```

Comment:
```suggestion
      className="answer-option d-flex flex-row justify-content-between flex-nowrap pb-2 pt-2"
```

Reply:
Can you also try to make every thing in the row centered aligned?

Reply:
I'll do that. Good point about the collapsible-card: Taking that away removes the outline, but the outline is not the one that's used in the mockups anyway. I would open a new issue to fix the outlines.